data/blog/2025/2025-02-03-summary-of-the-meeting-in-february-2025.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 2025-02-03-summary-of-the-meeting-in-february-2025
+author: bidi
+title: 'Summary of the meeting in February 2025'
+draft: false
+public: true
+created: '2025-02-04T11:00:00-01:00'
+updated: '2025-02-04T11:00:00-01:00'
+tags:
+    - meeting summary
+---
+
+On Monday, 3 February 2025, the Technical Steering Committee for the Laminas Project held its monthly meeting.
+The topics discussed included:
+
+- Creating a major release for [laminas-session](https://github.com/laminas/laminas-session).
+- Setting up email accounts for @getlaminas.org.
+- Dropping Support for PHP 8.1 in future releases for Laminas packages.
+- New design and layout for [getlaminas.org](getlaminas.org).
+
+<!--- EXTENDED -->
+# Creating a major release for laminas-session
+
+Backward compatibility among dependencies is desirable to maintain support for as many versions as is reasonable.
+Removing dependencies or limiting them to more recent versions has direct impact on all applications that use the older version.
+Releasing a backward incompatible version means some users may not be able to update to it without reworking the whole list of 3rd party packages.
+Eventually, releases that break compatibility are inevitable to benefit from new features, bugfixes and security updates.
+
+The TSC members agreed that a major release is needed for `laminas-session` because the update breaks backward compatibility.
```

Comment:
The component name is written once without and then with backticks. This should be standardised, therefore simply without.